### PR TITLE
Mimir 1605 publicationarchive h6fix

### PR DESCRIPTION
**Pull request by Carl-OW**

### CODE REVIEW

File: src/main/resources/assets/styles/_publicationArchive.scss [hunk 1]
```diff
@@ -10,6 +10,16 @@
     .publication-archive-body {
       & > * {
         max-width: 580px;
-      }  
+      }
+    }
+    .defineContentPhrase{
+      font-family: Roboto,sans-serif!important;
+      font-stretch: normal;
+      font-weight: 400;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_publicationArchive.scss:18]: There are two font-weight. Remove one.

File: src/main/resources/assets/styles/_publicationArchive.scss [hunk 2]
```diff
@@ -10,6 +10,16 @@
...
+      font-weight: 700;
+      margin: 0 0 0 0;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_publicationArchive.scss:20]: This line can be shortened to margin: 0;

File: src/main/resources/assets/styles/_publicationArchive.scss [hunk 3]
```diff
@@ -10,6 +10,16 @@
...
+      color: #162327;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_publicationArchive.scss:21]: No need for this line. The base color is already correct!

File: src/main/resources/assets/styles/_publicationArchive.scss [hunk 4]
```diff
@@ -10,6 +10,16 @@
...
+      font-size: 16px;
```
Comment (johnnadeluy) [inline src/main/resources/assets/styles/_publicationArchive.scss:22]: Maybe converting this to rem would be better? 16px is equals 1rem.

**Review by johnnadeluy — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/styles/_publicationArchive.scss (hunk 1 above)
Comment (Carl-OW) [reply src/main/resources/assets/styles/_publicationArchive.scss:18]: Removed 1 font-weight

File: src/main/resources/assets/styles/_publicationArchive.scss (hunk 2 above)
Comment (Carl-OW) [reply src/main/resources/assets/styles/_publicationArchive.scss:20]: done.

File: src/main/resources/assets/styles/_publicationArchive.scss (hunk 3 above)
Comment (Carl-OW) [reply src/main/resources/assets/styles/_publicationArchive.scss:21]: removed color: #162327;

File: src/main/resources/assets/styles/_publicationArchive.scss (hunk 4 above)
Comment (Carl-OW) [reply src/main/resources/assets/styles/_publicationArchive.scss:22]: changed all px to rem

**Review by johnnadeluy — APPROVED** [review]
Good job! 🌟

**Pull request merged**